README.md

```diff
@@ -1667,7 +1673,9 @@ arXiv, 2020.
 *[EMSGC: Event-based Motion Segmentation with Spatio-Temporal Graph Cuts](https://arxiv.org/pdf/2012.08730)*,  
 IEEE Trans. Neural Netw. Learn. Syst. (TNNLS), 2021.  [Project page](https://sites.google.com/view/emsgc),  [YouTube](https://youtu.be/ztUyNlKUwcM), [Code](https://github.com/HKUST-Aerial-Robotics/EMSGC.git).
 
-
+- <a name="WangUnEvMoseg"></a>Wang, Z., Guo, J., Daniilidis, K.,  
```

Comment:
typically the key is AuthorYYpublicationvenue, so Wang23arxiv  -> Wang23arxiva
We can edit this later, with an official publication.